### PR TITLE
data:Manoj-Kumar13

### PR DESCRIPTION
**Pull request by Manoj-Kumar13**
## Check List (Check all the applicable boxes) 

- [X] My code follows the code style of this project.
- [X] My change requires changes to the documentation.
- [X] I have updated the documentation accordingly.
- [X] All new and existing tests passed.
- [X] This PR does not contain plagiarized content.
- [X] The title of my pull request is a short description of the requested changes.

## Note to reviewers

This is my first pull request to create a profile on Linkfree and to get started with open-source.

<a href="https://gitpod.io/#https://github.com/EddieHubCommunity/LinkFree/pull/3669"></a>

### CODE REVIEW

File: data/Manoj-Kumar13.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+    {
+      "name": "Let's connect on LinkedIn",
+      "url": "https://www.linkedin.com/in/manoj-kumar13/",
+      "icon": "FaLinkedin"
+    },
+    {
+      "name": "Let's connect on GitHub",
+      "url": "https://github.com/Manoj-Kumar13",
+      "icon": "FaGithub"
+    }
```
Comment (Pal-Sandeep) [inline data/Manoj-Kumar13.json:22]: close the list

**Review by Pal-Sandeep — CHANGES_REQUESTED** [review]

File: data/Manoj-Kumar13.json (hunk 1 above)
Comment (Manoj-Kumar13) [reply data/Manoj-Kumar13.json:22]: done! please check now

**Review by Tejender1521 — APPROVED** [review]
Now Everything looks good

**Comment by Manoj-Kumar13** [comment]
> Now Everything looks good

When will it be merged?

**Review by Pradumnasaraf — APPROVED** [review]
Looks good to merge. Thank you for your contribution! :tada:

<p align="center"></p>

**Pull request merged**